cli/vm/cli.go

```diff
@@ -597,6 +606,22 @@ func handleBreak(c *cli.Context) error {
 	return nil
 }
 
+func handleRemoveBreak(c *cli.Context) error {
```

Comment:
It requires a unit-test, see `TestRun_WithNewVMContextAndBreakpoints` for reference.